ENIGMAsystem/SHELL/Universal_System/Extensions/MotionPlanning/motion_planning_struct.h

```diff
@@ -35,6 +35,14 @@ using std::multimap;
 #  error This file includes non-ENIGMA STL headers and should not be included from SHELLmain.
 #endif
 
+#ifdef COLLISION_NONE
+	#error The motion planning extension requires a collision system.
```

Comment:
It would be nice if new code didn't use tabs, but I suppose this isn't going to kill anyone.